MyWebPage/styles.css

```diff
@@ -16,3 +16,29 @@
       margin-right: 20%;
       font-family: 'Times New Roman', Times, serif;
   }
+
+  .title {
+    text-align: center;
+  }
+
+  .list {
+    text-align: center;
+    margin-left: 32%;
+    margin-right: 32%;
```

Comment:
Don't use percentage on this properties.
Another way to center content is:
`margin: 10px auto;`
this replace the line 26 & 27